flake.nix

```diff
@@ -7,6 +7,14 @@
 
   outputs = { self, nixpkgs }: {
 
+    nixosModules.default = {...}: {
+       fileSystems."/" = {
+         device = "/dev/sda1";
+         fsType = "ext4";
+       };
+       boot.loader.grub.device = "/dev/sda";
```

Comment:
Should we put this behind an `enable` flag? And that flag should be in a config namespace called `garnix`. So that you would do `garnix.enable = true;`. (We would have other `garnix.$whatever.enable` flags, e.g. for persistence, but since these are the options that we expect to be needed for *every* garnix hosted nixos config, it should be `garnix.enable`, we (@alexdavid & I) think.)